AutopanoPro.desktop

```diff
@@ -0,0 +1,13 @@
+[Desktop Entry]
+Categories=Application;Graphics
+Comment=Create panoramics
+Encoding=UTF-8
+Exec=AutopanoPro %u
+TryExec=AutopanoPro
+Icon=AutopanoPro.png
+MimeType=application/pano;
```

Comment:
this is actually an invalid mimetype.

Reply:
I am trying to define a new mime type for `*.pano` files, is that a bad idea? Should I remove it?